Documentation/configuration.md

```diff
@@ -68,7 +68,7 @@ To start etcd automatically using custom settings at startup in Linux, using a [
 + default: "default=http://localhost:2380,default=http://localhost:7001"
 
 ##### -initial-cluster-state
-+ Initial cluster state ("new" or "existing").
++ Initial cluster state ("new" or "existing"). For example, if you are statically bootstrapping 3 machines, use `new` on all of them.  For an existing cluster that you're [increasing the size of at runtime](runtime-configuration.md), use `existing`. If the wrong value is used, etcd will attempt to start and then stop.
```

Comment:
you can add a sentence after the first sentence to provide more explantation. I am just trying to avoid third pov.